label_studio/data_manager/functions.py

```diff
@@ -283,7 +283,18 @@ def get_all_columns(project, *_):
                 'explore': False,
                 'labeling': False
             }
-        }
+        },
+        {
+            'id': 'draft_exists',
+            'title': "Drafts",
+            'type': 'Boolean',
+            'help': 'True if draft exists for task',
```

Comment:
"True if at least one draft exists for the task"

Reply:
Fixed.